src/mod/acme/acme.go

```diff
@@ -292,12 +321,13 @@ func (a *ACMEHandler) ObtainCert(domains []string, certificateName string, email
 	}
 
 	// Save certificate's ACME info for renew usage
-	certInfo := &CertificateInfoJSON{
+	certInfo = &CertificateInfoJSON{
 		AcmeName:    caName,
 		AcmeUrl:     caUrl,
 		SkipTLS:     skipTLS,
 		UseDNS:      useDNS,
 		PropTimeout: propagationTimeout,
+		DNSServers:  strings.Split(dnsServers, ","),
```

Comment:
I am a bit worry that a space will exists between the `dnsServers` and comma, specially in cases where the user manually write the config file in vim (or other text editors)
e.g. `dns1.example.com, dns2.example.com`
I guess the better way to do so is iterate and clean the dnsServers entry one by one using something like

```
servers := strings.Split(dnsServers, ",")
    for i := range servers {
        servers[i] = strings.TrimSpace(servers[i])
    }
```